CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Change Log
 
-## [v1.2.1](https://github.com/ably/ably-python/tree/v1.2.1.)
+## [v1.3.0](https://github.com/ably/ably-python/tree/v1.3.0.)
 
-[Full Changelog](https://github.com/ably/ably-python/compare/v1.2.0...v1.2.1)
+[Full Changelog](https://github.com/ably/ably-python/compare/v1.2.0...v1.3.0)
+
+**Breaking API Change**: This release marks a semantically significant change in how the `ably-python` SDK's [SemVer](https://semver.org/) should be interpreted. Prior to version `1.3.0` the `ably-python` SDK's version number was overloaded to _broadly_ indicate the version of the Ably Protocol supported. This meant that both bug-fixes _and_ enhancements had to be shoehorned into the third (least significant) `SemVer` field whilst the first two fields reflected the Ably Protocol version. 
+
+With the `1.3.0` release we break away from this, there is a small enhancement when it comes to `1.2` Ably Protocol support with the addition of '[Channel Lifecycle Status](https://github.com/ably/ably-python/pull/276)'. 
```

Comment:
I would also remove this line. I don't feel it helps here.

Reply:
I added this line specifically because in numeric terms we're moving ahead of the Ably Spec and I wanted to re-iterate the point. 

Is it possible that an individual SDKs `README.md` is perhaps not the best forum to have this dialog with out users? By which I mean should we shelve the `1.3.0` release of `ably-python` for now, accumulate more spec point alignments on `main`, hold off on a `PyPi` release and publish an SDK agnostic blog posting about the change in `SemVer` semantics and then do the `1.3.0` release when we've accumulated enough 'value' to warrant the version bounce?